src/Content/LockException.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Kirby\Content;
```

Comment:
`Question:` Should this rather go into `Kirby\Exception`? We do have a namespace for exceptions, but we have started also to include some exceptions in other places.

Reply:
I thought about this as well and I cannot decide which way I like better.